io.catenax.material_flow_simulation_result/2.0.0/MaterialFlowSimulationResultAspect.ttl

```diff
@@ -0,0 +1,328 @@
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.0.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.0.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.0.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:io.catenax.material_flow_simulation_result:2.0.0#> .
+
+:MaterialFlowSimulationResultAspect a samm:Aspect ;
+   samm:preferredName "Material Flow Simulation Result Aspect"@en ;
+   samm:description "Aspect Model for the result of a Material Flow Simulation"@en ;
+   samm:properties ( :materialFlowSimulationResult ) ;
+   samm:operations ( ) ;
+   samm:events ( ) .
+
+:materialFlowSimulationResult a samm:Property ;
+   samm:preferredName "Material Flow Simulation Result"@en ;
+   samm:description "Contains standardized attributes for the result of a material flow simulation"@en ;
+   samm:characteristic :MaterialFlowSimulationResultCharacteristic .
+
+:MaterialFlowSimulationResultCharacteristic a samm:Characteristic ;
+   samm:preferredName "Material Flow Simulation Result Characteristic"@en ;
+   samm:description "Characteristic describing the aspect material flow simulation result"@en ;
+   samm:dataType :MaterialFlowSimulationResult .
+
+:MaterialFlowSimulationResult a samm:Entity ;
+   samm:preferredName "Material Flow Simulation Result"@en ;
+   samm:description "Result of a Material Flow Simulation"@en ;
+   samm:properties ( :description :runId :comment :expirationTimestamp :owner :dataQuality :shipments :timestamp ) .
+
+:description a samm:Property ;
+   samm:preferredName "Description"@en ;
+   samm:description "Verbal description"@en ;
+   samm:characteristic :Text ;
+   samm:exampleValue "Please simulate asap" .
+
+:runId a samm:Property ;
+   samm:preferredName "RunId"@en ;
+   samm:description "Id of the simulation run"@en ;
+   samm:characteristic :Text ;
+   samm:exampleValue "0fece48b-c8d1-4180-1a9caca6d67e" .
+
+:comment a samm:Property ;
+   samm:preferredName "Comment"@en ;
+   samm:description "Additional comments"@en ;
+   samm:characteristic :Text ;
+   samm:exampleValue "successful simulation " .
+
+:expirationTimestamp a samm:Property ;
+   samm:preferredName "ExpirationTimestamp"@en ;
+   samm:description "Date and time when the simulation is expired"@en ;
+   samm:characteristic samm-c:Timestamp ;
+   samm:exampleValue "2023-03-24T09:15:24.000Z"^^xsd:dateTime .
+
+:owner a samm:Property ;
+   samm:preferredName "Owner"@en ;
+   samm:description "Site identifier of the simulation originator"@en ;
+   samm:characteristic :OwnerCharacteristic .
+
+:dataQuality a samm:Property ;
+   samm:preferredName "Data Quality"@en ;
+   samm:description "Information on quality of simulation results"@en ;
+   samm:characteristic :DataQualityOptions ;
+   samm:exampleValue "0"^^xsd:positiveInteger .
```

Comment:
please change the datatype to `nonNegativeInteger` instead of `positiveInteger`

Reply:
has been changed to `nonNegativeInteger`